indico/modules/designer/client/js/index.js

```diff
@@ -240,11 +244,28 @@ import {$T} from 'indico/utils/i18n';
     var itemStyles = _.filter([item.bold ? 'bold' : null, item.italic ? 'italic' : null]);
 
     // Change the selectors so that they match the properties of the item
+    $('#text-color-selector').val(item.color);
+    var textColorPreview = $('#text-color-preview');
+    if (textColorPreview.hasClass('no-value') && item.color.length > 1) {
+      textColorPreview.removeClass('no-value');
+    }
+    textColorPreview.attr('style', 'background-color: ' + item.color + '; opacity: 1;');
+    if (item.color.length < 1) {
+      textColorPreview.addClass('no-value');
+    }
+    $('#bg-color-selector').val(item.background_color);
+    var bgColorPreview = $('#bg-color-preview');
+    if (bgColorPreview.hasClass('no-value') && item.background_color.length > 1) {
```

Comment:
`item.background_color` is null for existing badges. Maybe just check `item.background_color` which will cover both empty strings and null/undefined?

Also, why `> 1` and not `> 0` in this check?